ui/app/templates/workspace/projects/project/app/deployments.hbs

```diff
@@ -13,6 +23,20 @@
         from the CLI</p>
     </EmptyState>
   {{/each}}
+  {{#if (and this.hasMoreDeployments (not this.destroyed))}}
```

Comment:
Ah I see why this is at the controller. Maybe this could be a new component then. Not super tied to it just trying to keep some of this stuff out of controllers given the trend there.